src/styles.css

```diff
@@ -418,4 +413,17 @@ a.dropdown-item:hover {
   .desc-text{
     font-size: 0.95rem;
   }
+
+  .label, .is-small-font {
```

Comment:
What is is small font?

Reply:
This class is being used in `configuration-group` component where we need same css property but that is not label. I can remove it from here and use it at component level.

Reply:
sure. and use label instead of .label? (tag vs class)